app/views/comments/_form.html.erb

```diff
@@ -11,22 +11,25 @@
     </div>
   <% end %>
 
-  <div>
+  <div class='comments_form'>
+    <div>
     <%= form.label :author_id, style: "display: block" %>
-    <%= form.text_field :author_id %>
-  </div>
+    <%= form.text_field :author_id ,placeholder: 'Author id'%>
+    </div>
 
-  <div>
+   <div>
     <%= form.label :post_id, style: "display: block" %>
-    <%= form.text_field :post_id %>
-  </div>
+    <%= form.text_field :post_id , placeholder: 'Post id'%>
+   </div>
 
-  <div>
+   <div>
     <%= form.label :text, style: "display: block" %>
     <%= form.text_area :text %>
-  </div>
+   </div>
 
-  <div>
+   <div>
     <%= form.submit %>
-  </div>
+   </div>
+
+   </div>
 <% end %>
```

Comment:
- [ ] From the `Post#index` page, I am not able to access the `Comment's form` or the `Post#show` page. I have to manually input the URL for the `Comment's form` or the `Post#show` page to be able to access the form. Kindly create a link in the `Post#index` form so that it is easy for a user to access the `Comment's form`. 👍 😎